docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -108,7 +108,15 @@ If you are working in the platform run `SUB_BUILD=PLATFORM ./gradlew format` fro
 
 ### Connector
 
-If you are working on an individual connectors run: `./gradlew :airbyte-integrations:<directory the connector is in e.g. source-postgres>:format`.
+If you are working on an individual connectors run: 
```

Comment:
This only works for python connectors but not for Java which is the `:format`

Reply:
Hey, Marcos. Can you run `./gradlew :airbyte-integrations:connectors:source-mysql:format` in my local environment? 

```
Task 'format' not found in project ':airbyte-integrations:connectors:source-mysql'.
```